loguru/_logger.py

```diff
@@ -1378,6 +1378,7 @@ def opt(
             warnings.warn(
                 "The 'ansi' parameter is deprecated, please use 'colors' instead",
                 DeprecationWarning,
+                stacklevel=1,
```

Comment:
Given than [`B028`](https://beta.ruff.rs/docs/rules/no-explicit-stacklevel/) considers the default `stacklevel=1` as bad, shouldn't we replace it with `stacklevel=2`?

It seems indeed best practice, I didn't realize that the warnings weren't even visible to the user by default...

It slightly changes the behavior but this is for the better in my opinion.

Reply:
Sure thing. Done.